frontend/templates/views/partials/code-preview/code-preview.j2

```diff
@@ -0,0 +1,12 @@
+{% if preview.mode == 'inline' %}
```

Comment:
Does it make sense to use this in the examples template as well? //cc @matthiasrohmer

Reply:
I think we should look at that when we have implemented the playground link and iframe preview, but my gut feeling is that there will be enough differences to keep it separated.